### ISSUE TITLE
checkm-genome

### ISSUE THREAD

**Issue by nick-youngblut** [bioconda/bioconda-recipes#5967]
I installed ` checkm-genome:     1.0.7-py36_0       bioconda` via `conda install -c bioconda checkm-genome`. When running checkm, I get the following error:

~~~
  File "/ebio/abt3_projects/software/tmp/miniconda3/envs/llga/bin/checkm", line 58
    print ''
           ^
~~~

Should `checkm-genome 1.0.7-py36_0` exist? The python code is written in py2.7, not py3.6

**Comment by druvus** [comment]
You're right, it is py27 only. See #5978 for a fixing to not build py3k versions. 

@bgruening could you remove non-working py3k builds of checkm-genome from anaconda?

**Comment by alienzj** [comment]
Now CheckM support python3.
